packages/bot-web-ui/src/components/quick-strategy/inputs/__tests__/qs-checkbox.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { act, fireEvent, render, screen } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import userEvent from '@testing-library/user-event';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import QSCheckbox from '../qs-checkbox';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
```

Comment:
i guess we can remove these mocks since test can be passed without them
```suggestion
jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
```
<img width="982" alt="Screenshot 2023-11-02 at 11 07 14" src="https://github.com/binary-com/deriv-app/assets/103181650/21b91a6d-bf61-4be7-990b-ab389315e101">

Reply:
the same thing can be done also for the test-files `qs-input-label.spec.tsx`, `qs-input.spec.tsx`

Reply:
maybe at the same time could be removed line:
`jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());`
in the file `symbol.spec.tsx`?

Reply:
done